core/src/main/java/com/rudderstack/android/sdk/core/RudderContext.java

```diff
@@ -185,4 +185,9 @@ private boolean getAmazonFireAdvertisingID() throws Exception {
         return true;
     }
 
+    public String getAdvertisingId() {
+        if (this.deviceInfo == null) { return null; }
+        return this.deviceInfo.getAdvertisingId();
+    }
+
```

Comment:
Expose the `adTrackingEnabled` as well?

Reply:
done

Reply:
```suggestion
@Nullable public String get...
```

Check : https://github.com/rudderlabs/rudder-sdk-android/blob/07484c210c252b3c07320654a7185155e3cf8238/core/src/main/java/com/rudderstack/android/sdk/core/RudderClient.java#L40